### PR TITLE
SPDX exceptions

### PR DESCRIPTION
**Pull request by mnonnenmacher**
Please see the commit messages for details.

---
This change is [Reviewable](https://reviewable.io/reviews/heremaps/oss-review-toolkit/1073)

### CODE REVIEW

File: model/build.gradle [hunk 1]
```diff
@@ -75,9 +75,71 @@ task generateSpdxLicenseList {
     }
 }
 
+task generateSpdxLicenseExceptionList {
+    doLast {
+        println "Fetching SPDX license list..."
```
Comment (sschuberth) [inline model/build.gradle]: "license exception list"

File: model/build.gradle [hunk 2]
```diff
@@ -75,9 +75,71 @@ task generateSpdxLicenseList {
...
+             * limitations under the License.
+             *
+             * SPDX-License-Identifier: Apache-2.0
+             * License-Filename: LICENSE
+             */
+
+            package com.here.ort.model.config
+
+            /**
+             * An enum containing all SPDX license IDs. This class is generated by the Gradle task
```
Comment (sschuberth) [inline model/build.gradle]: "license exception IDs"

File: model/build.gradle [hunk 3]
```diff
@@ -75,9 +75,71 @@ task generateSpdxLicenseList {
...
+        def json = jsonSlurper.parse(spdxUrl)
+        def spdxLicenseExceptionIds = json.exceptions.collect {
+            it["licenseExceptionId"]
+        }
+
+        println "Found ${spdxLicenseExceptionIds.size()} SPDX license exception identifiers."
+
+        def licenseExceptionEnumFile = file('src/main/kotlin/config/SpdxLicenseException.kt')
+        licenseExceptionEnumFile.write("""\
+            /*
```
Comment (sschuberth) [inline model/build.gradle]: Note to myself: In a later commit we could move this to a function that also replaces "2018" with the current year.

File: model/build.gradle (hunk 1 above)
Comment (mnonnenmacher) [reply model/build.gradle]: Done.

File: model/build.gradle (hunk 2 above)
Comment (mnonnenmacher) [reply model/build.gradle]: Done.

File: model/build.gradle (hunk 3 above)
Comment (mnonnenmacher) [reply model/build.gradle]: Yes, this also needs more cleanup as there is also too much duplication currently, but as there are more SPDX related changes coming I first wanted to see watch other changes are required before optimizing the code.

**Pull request merged**